app/models/document_type/primary_publishing_organisation_field.rb

```diff
@@ -8,7 +8,8 @@ def payload(edition)
   end
 
   def updater_params(_edition, params)
-    { primary_publishing_organisation: params[:primary_publishing_organisation] }
+    value = params[id] && params[id].map(&:presence).compact
+    { id.to_sym => value }.compact
```

Comment:
Maybe not for today - but could this be easier if the form didn't submit an array?

Reply:
Well, we store it as an array, so probably not on the whole. It's another one of those situations where we (can) only partially trust the submitted params.